Gemfile

```diff
@@ -1,46 +1,55 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
```

Comment:
- [ ] **Optional:** Kindly confirm that the app is working correctly when cloned as I encountered the error below when I attempted to run the code locally.

![image](https://user-images.githubusercontent.com/27709832/198107410-e0ff32ef-a27e-4c96-ad8a-db6b90223527.png)